### ISSUE TITLE
Love your work, I use the editor in all my projects!!

### ISSUE THREAD

**Issue by artknight** [slab/quill#4365]
Hey, sorry to post it here as I did not see any way to contact you. I really like your work, so I am going to feature it in the tomorrow's issue of my my daily newsletter,  https://dailysandbox.com

**Comment by MagisterUnivers** [comment]
Did you managed to solve HTML to markdown transform in correct way? The library itself saving HTML, which is weird. 

I guess its better to save result in markdown, and then parse it on front-end, but convertation from library is incorrect, because its not actually apply something, its applying own classes, which not passed to markdown and any result you can take.

![image](https://github.com/user-attachments/assets/1add003a-50e2-4349-92c9-7dee63fe0bfb)


I am just wonder how you do that. Because i cant take something good on frontend

**Comment by artknight** [comment]
@MagisterUnivers The Quill Editor uses its own **delta format** to store the content. By default, it is not supposed to convert it to markdown.

**Comment by MagisterUnivers** [comment]
@artknight You are absolutely right. I just left it work with HTML format and dangerouslyInsertHTML + DomPufiry. It is enough.

But if you want simple, fast, component based library for text-editor and markdown - **_use MDXEditor_**. I refactored it to that library, and i am so satisfied.
